src/wx/gfxviewers.cpp

```diff
@@ -1233,11 +1241,13 @@ class PaletteViewer : public Viewer {
     }
     void SelBG(wxMouseEvent& ev)
     {
+	(void)ev;
```

Comment:
This also I'd rather just turn off, unless there is an actual problem.

Reply:
For `wx*` functions, I think there could be a case of  just using the parent class function. Not sure about this one, though.